Fix #3060 Next Cloud handle special ingredients ##

ellezhu1 opened the pull request:
Fix #3060

Before and After:

![nextCloudBefore1](https://github.com/TandoorRecipes/recipes/assets/89816058/a094021f-2607-47b9-9309-95703e7f4333)
![nextCloudAfter](https://github.com/TandoorRecipes/recipes/assets/89816058/e69d531f-5ed5-4702-91a3-f014d988a57a)

vabene1111 commented:
thanks for the RP, two ideas:

1. should we use `.startwith()` to make sure lines that just contain a double ## somehwere are not ignored
2. what do you think of instead of ignoring the lines adding them as a header in tandoor (by setting `is_header` to true on the ingredient and setting the text (without the ##) into the `note` field (leaving food/unit None and amount 0)

ellezhu1 commented:
1. I didn't think it was very likely; however, you make a good point. I have gone ahead and changed it to use `.startswith()` 
2. I did consider making them into subheaders but I couldn't figure out how to do it. Your tip about the `note` field was very helpful. I have now implemented the lines as headers.

Thanks for the feedback!

vabene1111 commented:
perfect, thank you. Will get both PR's merged for the next release.
